totalRP3/Modules/Register/Characters/RegisterAbout.lua

```diff
@@ -335,11 +344,19 @@ function refreshTemplate2EditDisplay()
 		_G[frame:GetName().."Bkg"]:SetSelectedIndex(frameData.BK or 1);
 		_G[frame:GetName().."TextScrollText"]:SetText(frameData.TX or "");
 		setupIconButton(_G[frame:GetName().."Icon"], frameData.IC or TRP3_InterfaceIcons.Default);
-		_G[frame:GetName().."Icon"]:SetScript("OnClick", function()
-			showIconBrowser(function(icon)
-				frame.frameData.IC = icon;
-				setupIconButton(_G[frame:GetName().."Icon"], icon);
-			end, frameData.IC);
+		_G[frame:GetName().."Icon"]:SetScript("onMouseDown", function(self, button)
+			if button == "LeftButton" then
+				showIconBrowser(function(icon)
+					frame.frameData.IC = icon;
+					setupIconButton(_G[frame:GetName().."Icon"], icon);
+				end, frameData.IC);
+			elseif button == "RightButton" then
+				TRP3_MenuUtil.CreateContextMenu(self, function(_, description)
+					description:CreateButton(loc.UI_ICON_COPY, function() TRP3_API.SetLastCopiedIcon(frameData.IC); end);
+					description:CreateButton(loc.UI_ICON_COPYNAME, function() TRP3_API.popup.showCopyDropdownPopup({frameData.IC or TRP3_InterfaceIcons.Default}); end);
```

Comment:
There's a few places where COPY doesn't have a nil check but COPYNAME does. As far as I can tell, these are all places where there always is an icon set by default, so the COPY version makes as much sense as the COPYNAME one, but they should probably be consistent™.